src/NServiceBus.AzureStoragePersistence/Timeout/AzureTimeoutStorageDefaults.cs

```diff
@@ -11,5 +11,6 @@ public class AzureTimeoutStorageDefaults
         public const int CatchUpInterval = 3600;
         public const string PartitionKeyScope = "yyyyMMddHH";
         public const bool CreateSchema = true;
+        public const string TimeoutStateBlobName = "timeoutstate";
```

Comment:
:+1: as we've discussed, need to make sure we put this on the list of things to document